libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionView.java

```diff
@@ -814,15 +814,15 @@ private void updateLandscapeConstraintsTo(int layoutRes) {
    * @param model to provide the current steps and unit type
    */
   private void updateInstructionList(InstructionModel model) {
-    instructionListAdapter.updateSteps(model.getProgress());
+    instructionListAdapter.updateSteps(getContext(), model.getProgress(), locale, unitType);
```

Comment:
Was the `locale` and `unitType` moved here so they can be updated on the fly and the list will reflect the latest changes?

Reply:
@danesfeder yes because otherwise, the locale was being set in the adapter when it was initialized, which was before the locale was actually set in `InstructionView`